docs/WindowsBuild.md

```diff
@@ -138,27 +137,25 @@ path S:\b\llvm\bin;%PATH%
 ```
 
 ### 6. Build CMark
-- This must be done from within a developer command prompt. CMark is a fairly
-  small project and should only take a few minutes to build.
+- CMark is a fairly small project and should only take a minute to build.
```

Comment:
I really do dislike the estimates since they don't really tell you much.  If anything, using the custom ninja format like we do in swift is more useful.